src/components/shared/thread/SampleGraph.js

```diff
@@ -50,7 +50,6 @@ export class ThreadSampleGraphImpl extends PureComponent<Props> {
   _canvas: null | HTMLCanvasElement = null;
   _takeCanvasRef = (canvas: HTMLCanvasElement | null) =>
     (this._canvas = canvas);
-  _resizeListener = () => this.forceUpdate();
 
   _renderCanvas() {
     const canvas = this._canvas;
```

Comment:
It strikes me that there's no requestAnimationFrame for this, is that normal? (not for this PR)

Reply:
Hmm, we don't have one in the `ActivityGraphCanvas` component either. I'm not sure if we should have. Also I see that we have a fAF inside the WithSize component. Maybe it's fine but I will check still.